src/Data/Geometry/Structure/Feature.hs

```diff
@@ -179,6 +182,7 @@ data ScenarioProperties = ScenarioProperties
     , defaultActiveColor :: !HexColor
     , defaultStaticColor :: !HexColor
     , defaultLineColor   :: !HexColor
+    , hiddenProperties   :: !(Maybe (JS.Array JSString))
```

Comment:
Could you change that to `[JSString]`? Default then is empty list instead of Nothing.